### PR TITLE
MIMIR-73: Use title from tbprocessor data if it exists

### PR DESCRIPTION
**Pull request by itemodahl**
Move code to lib

MIMIR-73

### CODE REVIEW

File: src/main/resources/lib/ssb/table.ts [hunk 1]
```diff
@@ -9,24 +9,25 @@ import { Dataset as JSDataset } from '../types/jsonstat-toolkit'
...
-} = __non_webpack_require__( '/lib/ssb/dataset/dataset')
-const {
-  fromDatasetRepoCache
+  datasetOrUndefined
 } = __non_webpack_require__('/lib/ssb/cache')
 const {
   fetchStatbankSavedData
 }: StatbankSavedLib = __non_webpack_require__('/lib/ssb/dataset/statbankSaved')
 
+const util: UtilLibrary = __non_webpack_require__( '/lib/util')
```
Comment (Garlov) [inline src/main/resources/lib/ssb/table.ts:28]: linting: unused

File: src/main/resources/lib/ssb/table.ts [hunk 2]
```diff
@@ -9,24 +9,25 @@ import { Dataset as JSDataset } from '../types/jsonstat-toolkit'
...
 } = __non_webpack_require__( '/lib/util')
+
 const {
-  getDataset,
-  extractKey
-} = __non_webpack_require__( '/lib/ssb/dataset/dataset')
-const {
-  fromDatasetRepoCache
+  datasetOrUndefined
 } = __non_webpack_require__('/lib/ssb/cache')
```
Comment (Garlov) [inline src/main/resources/lib/ssb/table.ts:23]: SSBCacheLibrary interface

**Review by Garlov — CHANGES_REQUESTED** [review]

File: src/main/resources/lib/ssb/keyFigure.ts [hunk 3]
```diff
@@ -30,13 +30,9 @@ const {
   getImageCaption
 } = __non_webpack_require__( '/lib/ssb/utils')
 const {
-  fromDatasetRepoCache
+  datasetOrUndefined
 } = __non_webpack_require__('/lib/ssb/cache')
```
[the remark this answers is not in the document]
Comment (itemodahl) [reply src/main/resources/lib/ssb/keyFigure.ts]: added

File: src/main/resources/lib/ssb/table.ts (hunk 2 above)
Comment (itemodahl) [reply src/main/resources/lib/ssb/table.ts]: added

File: src/main/resources/lib/ssb/table.ts (hunk 1 above)
Comment (itemodahl) [reply src/main/resources/lib/ssb/table.ts]: removed

**Review by Garlov — APPROVED** [review]
Looks good 👍

**Pull request merged**